definitions/npm/jest-useragent-mock_v0.1.x/flow_v0.83.x-/jest-useragent-mock_v0.1.x.js

```diff
@@ -0,0 +1,5 @@
+declare module 'jest-useragent-mock' {
+  declare export function mockUserAgent(userAgent?: string): void;
```

Comment:
It's `string | null` in the TS definition, any reason to diverge?

Reply:
Because when I read into the [source code](https://github.com/ariesjia/jest-useragent-mock/blob/master/src/index.js#L15). Allowing null has no real value. User agent should always be a string, allowing empty value is just a means of a fallback but extending it to support `null` feels wrong. In fact you can probably pass in number or really anything and it still works but ideally it should only be a string.

implicitly leaving out the value seems ok, explicitly passing in an empty value such as `null` isn't useful and ideally shouldn't be allowed. Although `fakeUserAgent` is initialised as `null` it does still default the return value as `''`.

Reply:
https://github.com/ariesjia/jest-useragent-mock/issues/5

I understand your reasoning, but diverging from the TS typings (if any) makes maintaining the flow typed one much tedious and confusing.